docs/hub/datasets-download-stats.md

```diff
@@ -2,7 +2,11 @@
 
 ## How are download stats generated for datasets?
```

Comment:
would be clearer maybe to word it like: "How are downloads counted for datasets" (same for models)